nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/AbstractKudu.java

```diff
@@ -94,6 +95,27 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    protected static final PropertyDescriptor FLUSH_MODE = new PropertyDescriptor.Builder()
+            .name("Flush Mode")
+            .description("Set the new flush mode for a kudu session\n" +
```

Comment:
Would it be cleaner to define AllowableValue objects and set the description of each allowable value instead of doing it in the property description? (example in AbstractPutHBase). It's just a general question... I don't have a strong opinion on this.

Reply:
@pvillard31 , different methods, archiving the same purpose :-) I don't have any opinion. It was they way Ricky suggested in its initial PR #2020 . I would leave it like that since it looks straightforward :-)

Reply:
My only concern is: if we update the Kudu dependency and a new SessionConfiguration.FlushMode option is made available in the library, the contributor bumping the Kudu version will have to remember to update the description otherwise there will be an undocumented option. But I'm fine with this approach: it does not make a huge difference.